ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -483,6 +486,12 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         bottom.priority = .defaultHigh
 
         NSLayoutConstraint.activate([leading, trailing, top, bottom])
+
+        if #available(iOS 15.0, *) {
+            // These are consistent with UIKit's default navigation bar
```

Comment:
Same question as the other comment - do these values align with what design wants NavBar to look like? Consistency with UIKit is fine, but it's more important for us to be consistent across Fluent 2 designs.

Reply:
Design hasn't given us any specs for text sizes in any content size categories besides [`.large`](https://developer.apple.com/documentation/uikit/uicontentsizecategory/1622934-large), so keeping consistency with UIKit is the best we can do.

Reply:
this is completely different direction than what we had before. We usually cap and min based on font size not the size category because most of our padding is fixed and not tokenize. i am not sure we should set min and max category necessarily.

Reply:
I think this direction is worth pursuing. Peeking into a `UIView`'s [`appliedContentSizeCategoryLimits`](https://developer.apple.com/documentation/uikit/uiview/3750922-appliedcontentsizecategorylimits) property, we can see that UIKit's navigation bar has category caps set accordingly. Plus, this is arguably better than the alternative of specifying a "magic number" of 21pt somewhere.